src/util/strip.ts

```diff
@@ -12,6 +12,8 @@ type StripOptions = { preserveFormatting?: boolean; preventTrim?: boolean; strip
 
 const regexSpanTagOnlyContainsWhitespaces = /<span[^>]*>([\s]+)<\/span>/gim
 
+const regexExpForEmptyFormattingTags = /<[^/>][^>]*> *<\/[^>]+>/gim
```

Comment:
I would change the space to any whitespace to handled cases like `<b>\n\n\n</b>`